src/containers/Masthead/components/MastheadSearchForm.jsx

```diff
@@ -101,6 +105,22 @@ export class MastheadSearchForm extends Component {
     }
   }
 
+  async handleTaxonomyId(taxId) {
+    const { history } = this.props;
+    let taxonomyFunction = fetchTopic;
+    if (taxId.includes('urn:resource:')) {
+      taxonomyFunction = fetchResource;
+    }
+    try {
+      const taxElement = await taxonomyFunction(taxId);
```

Comment:
Jeg ser at både `fetchResource` og `fetchTopic` tar to parametere `(id, locale)`, trenger du ikke å sende med `locale`?

Reply:
nja, keep in mind at dette er verdens beste språk javascript, så locale vil bare ende opp med å være `undefined` om du ikke sender det med, så det vil nok fungere, men kanskje greit å gjøre så de to funksjonene fungerer på samme måte uansett?